main.cpp

```diff
@@ -188,214 +286,313 @@ struct Range
+    if(takeOff) 
+        return !retractLandingGear;
+    return retractLandingGear; 
+}
+
 struct PlaneTail 
 {
-    // 1) weight of lower rudder (float)
     float weightLowerRudder =  2400.1f;
-    // 2) length of lower rudder (float )
     float lengthOfLowerRudder = 421.5f;
-    // 3) things its auxiliary power unit can do (std::string)
     std::string auxPower = "helps plane turn";
-    // 4) weight of upper rudder (float)
     float weigtUpperRudder = 2400.1f;
-    // 5) height of upper rudder (float)
     float heightUpperRudder = 421.5f;
-
-    // 1) its small engine powers the jet's electrical when on the ground
-    void runSmallEngine (bool electricalsOn = true);
-    // 2) its power unit powers the air-conditioning systems when on the ground
-    void runPowerUnit (bool airConditioningOn = true);
-    // 3) its rudder section turns the airplane from right to left in flight
+    int powerConsumed = 1;
+    
+    void showAlertMsg (bool);
+    void consumePower (bool);
     void turnPlane (bool lowerRudderOpen = true);
 };
-/*
-Thing 8) passenger cabin
- */
+
+void alert()
+{
+    std::cout << "Electricals are running" << std::endl;
+}
+
+void PlaneTail::showAlertMsg (bool switchOn)
+{ 
+    // when the switch is on, alert to pilot electicals are running
+    if(switchOn)
+        alert();
+    std::cout << "Electricals are not running" << std::endl;
+}
+
+void PlaneTail::consumePower (bool engineRunning)
+{
+    while (engineRunning) {
+        std::cout << "Engine running" << "\n";
+        powerConsumed++;
+    }
+    
+}
+
+void PlaneTail::turnPlane (bool lowerRudderOpen)
+{
+    bool right;
+    if(lowerRudderOpen) right = true; 
+    
+}
+
 struct PassengerCabin
 {
-    // 1) number of seats (int)
     int numSeats = 416;
-    // 2) material of seat cover (std::string)
     std::string seatMaterial = "leather";
-    // 3) number of toilets (int)
     int numToilets = 4;
-    // 4) weight of carry-on luggage (float)
     float weightCarryOn = 1;
-    // 5) aisle width (float)
     float aisleWidth = 42.1f;
 
-    // 1) carry passengers
     void carryPassengers (int numPassengers = 200);
-    // 2) carry toilets
-    void carryToietls (int numOfToilets);
-    // 3) carry 
-    void carryBlankets (int numPassengers, int numBlanketsPerPassenger);
+    void carryToilets (int numOfToilets);
+    void blanketsAlert (int numPassengers, int numBlanketsPerPassenger);
 };
+using namespace std;
+void PassengerCabin::carryPassengers (int numPassengers)
+{
+    cout << "Carrying: " << numPassengers << "passengers"<<endl;
+}
+void PassengerCabin::carryToilets (int numOfToilets)
+{
+    cout << "Carrying: " << numOfToilets << "toilets"<<endl;  
+}
+
+void PassengerCabin::blanketsAlert (int numPassengers, int numBlankets)
+{
+    if(numPassengers < numBlankets)
+    {
+    cout << "Carrying: " << numBlankets << "blankets"<<endl;
+    };
+    cout << "Stock " << (numPassengers - numBlankets) << "more blankets at least"<<endl;
+}
```

Comment:
Fix the indentation
add space around `<<`
do not put semicolons at the end of curly brace blocks unless you're defining a structure or lambda (covered later)

- [ ] revise